handler.go

```diff
@@ -465,6 +465,7 @@ func (fm *Frontman) Run(inputFilePath string, outputFile *os.File, interrupt cha
 		input, err := fm.FetchInput(inputFilePath)
 		if err != nil && err == ErrorMissingHubOrInput {
 			log.Warnln(err)
+			time.Sleep(10 * time.Second)
```

Comment:
Please add a comment why we need a sleep here.
I guess this can easily be forgotten and then someone might removes it later